static/versionDetails.json

```diff
@@ -417,6 +417,11 @@
 			"flag": "shouldEnablePurgeNodes",
 			"description": "Automatically delete and cleanup unreachable nodes",
 			"type": "boolean"
+		},
+		{
+			"flag": "autoUpgradeSchedule",
+			"description": "Schedule to automatically check for and install updates",
```

Comment:
```suggestion
			"description": "Schedule to automatically check for and install updates, when `auto-upgrades-enabled` is used",
```

Reply:
Not quite sure about this suggestion, but people could think all they need to do is set that value, so I want to convey that both pieces are needed.